SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/EntityComponentUpdateRecordTest.cpp

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+
+#include "SpatialView/EntityComponentUpdateRecord.h"
+
+#include "EntityComponentTestUtils.h"
+
+#define ENTITYCOMPONENTUPDATERECORD_TEST(TestName) \
+	GDK_TEST(Core, EntityComponentUpdateRecord, TestName)
+
+using namespace SpatialGDK;
```

Comment:
Can't remember if we like "using namespace"?

Reply:
There are split opinions however it is in the codebase. (Personally I don't like it)
Can we put the tests in the namespace though? Then we get around it.

Reply:
Removed `using`, moved test code to SpatialGDK namespace instead.